documentation/how-to/digitize.en.md

```diff
@@ -55,6 +55,10 @@ You can use the volume keys for adding or removing vertices while in digitizing
 !!! note
     This feature is available on Android only.
 
+Additionally Finger Tap Digitizing Mode in QField, offer to users a swift method to add vertices by tapping on the canvas, balancing speed and functionality. Accessed through the settings panel, this mode caters to scenarios where rapid data input is crucial.
```

Comment:
I believe we try to use "you" instead of "users" throughout our documentation.

```suggestion
Additionally, QField has a _finger tap_ digitizing mode where vertices are added by tapping on the canvas. Activated through the settings panel, this mode caters to scenarios where rapid data input is crucial.
```

Reply:
Completely agree!